hack/jenkins/e2e-runner.sh

```diff
@@ -207,6 +207,7 @@ function setup_gci_vars() {
 if running_in_docker; then
     record_command "${STAGE_PRE}" "download_gcloud" curl -fsSL --retry 3 --keepalive-time 2 -o "${WORKSPACE}/google-cloud-sdk.tar.gz" 'https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz'
     install_google_cloud_sdk_tarball "${WORKSPACE}/google-cloud-sdk.tar.gz" /
+    gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
I want complex logic like this in bash, not jenkins (I will be much happier when we yank that activate-gce-service-account builder).

In the mean time, there is no harm in activating the same account multiple times.